File: from-3201-to-3300/3295-report-spam-message/solution.go
func reportSpam(message []string, bannedWords []string) bool {
    banned := make(map[string]bool)
    for _, word := range bannedWords {
        banned[word] = true
    }
    banCount := 0
    for _, word := range message {
        if banned[word] {
            banCount += 1
        }
        if banCount == 2 {
            return true
        }
    }
    return false
}

File: from-3201-to-3300/3295-report-spam-message/solution.py
class Solution:
    def reportSpam(self, message: List[str], bannedWords: List[str]) -> bool:
        banned = {word : True for word in bannedWords}
        # banned = set(bannedWords)
        banCount = 0
        for word in message:
            if banned.get(word, False):
            # if word in banned:
                banCount += 1
            if banCount == 2:
                return True
        return False